test/persistence.jl

```diff
@@ -69,4 +69,16 @@ gs = load(joinpath(testdir, "testdata", "twographs.dot"), :dot)
 @test save(f, p1, :gexf) == 1
 @test_throws ErrorException load(STDIN, :gexf)
 
+#test :net
+g10 = CompleteGraph(10)
+@test save("testg10.net", g10, :net) == 1
```

Comment:
Create a temp file here, please. Don't assume you have write permissions in whatever directory we happen to be in.